src/lib.rs

```diff
@@ -1,82 +1,100 @@
-//! This crate contains checked implementations of `transmute()`.
+//! This crate contains checked implementations of transmutation procedures, some of which
+//! ensure memory safety.
+//! 
+//! ## Crate outline
+//! 
+//! The following modules are available:
+//! 
+//! - The functions in the [`base`](base/index.html) module are not inherently
+//! safe, but just protected against out of boundary access (like trying to
+//! create an 8-byte type from 7 bytes). These functions are as safe as
+//! the data passed to them: any attempt of transmuting data to an invalid
+//! memory representation is still undefined behavior. Moreover, unaligned
+//! memory access is not prevented, and must be previously ensured by the
+//! caller.
+//! - The [`guard`] module contains the **Guard** API, which imposes slice
```

Comment:
🤔 not sure about the `<strong>` placement here; does it work better around both Guard and API? up to you but just wanted to point this out